ferrocene/tools/traceability-matrix/README.md

```diff
@@ -17,6 +17,8 @@ On the one hand, we have the Ferrocene Language Specification (FLS). Simply spea
 
 On the other hand we have tests. We need to find at least one test for each section from the FLS. This test is then annotated with the id of the statement. This annotating is happening through a comment which is getting added to the end of the file. It is possible to annotate one test with multiple ids.
```

Comment:
```suggestion
On the other hand we have tests. We need to find at least one test for each section from the FLS. This test is then annotated with the id of the section. This annotating is happening through a comment which is getting added to the end of the file. It is possible to annotate one test with multiple ids.
```

Reply:
Isn't it that we need at least one test for each statement? In theory we could only use the ids of the statements and never a section id.

The "advantage" of using the section ids is just that it covers a set of statements.

Reply:
that is what was initially thought, but it so happens that that granularity is not needed for qualification... we were even thinking of removing that functionality from traceability-matrix

Reply:
Changed it to section